WS2012R2/lisa/remote-scripts/ica/Kdump_Results.sh

```diff
@@ -19,32 +19,27 @@
 # permissions and limitations under the License.
 #
 #####################################################################
-LinuxRelease()
-{
-    DISTRO=`grep -ihs "buntu\|Suse\|Fedora\|Debian\|CentOS\|Red Hat Enterprise Linux" /etc/{issue,*release,*version}`
 
-    case $DISTRO in
-        *buntu*)
-            echo "UBUNTU";;
-        Fedora*)
-            echo "FEDORA";;
-        CentOS*)
-            echo "CENTOS";;
-        *SUSE*)
-            echo "SLES";;
-        *Red*Hat*)
-            echo "RHEL";;
-        Debian*)
-            echo "DEBIAN";;
-    esac
-}
+dos2unix utils.sh
 
-LogMsg()
-{
-    # To add the time-stamp to the log file
-    echo `date "+%a %b %d %T %Y"` ": ${1}"
+#
+# Source utils.sh to get more utils
+# Get $DISTRO, LogMsg directly from utils.sh
+#
+. utils.sh || {
+	echo "Error: unable to source utils.sh!"
+	exit 1
 }
 
+#
+# Source constants file and initialize most common variables
+#
+UtilsInit
+
+ICA_TESTFAILED="TestFailed"
```

Comment:
Will operate them in kdump_config.sh, kdump_execute.sh kdump_resutls.sh.

That's great!